internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -220,10 +220,20 @@ func (c *ClusterManager) processAcceptedClusters() []error {
 	for i := range acceptedClusters {
 		cluster := acceptedClusters[i]
 		glog.V(10).Infof("accepted cluster ClusterID = %s", cluster.ClusterID)
-		metrics.UpdateClusterStatusSinceCreatedMetric(cluster, api.ClusterAccepted)
-		if err := c.reconcileAcceptedCluster(&cluster); err != nil {
-			errs = append(errs, errors.Wrapf(err, "failed to reconcile accepted cluster %s", cluster.ID))
-			continue
+		if cluster.ClusterType != api.Enterprise.String() {
+			metrics.UpdateClusterStatusSinceCreatedMetric(cluster, api.ClusterAccepted)
+			if err := c.reconcileAcceptedCluster(&cluster); err != nil {
+				errs = append(errs, errors.Wrapf(err, "failed to reconcile accepted cluster %s", cluster.ID))
+				continue
+			}
+		} else {
+			cluster.Status = api.ClusterStatus(api.ClusterProvisioning.String())
+			updateErr := c.ClusterService.Update(cluster)
+			if updateErr != nil {
+				errs = append(errs, errors.Wrapf(updateErr, "failed to update status of accepted %s cluster %s", api.Enterprise.String(), cluster.ID))
+				continue
```

Comment:
is the `continue` here just to skip the log in case of errors? If yes, I believe an `else` would be cleaner